modules/dashboard/test/DashboardTest.php

```diff
@@ -41,5 +41,300 @@ public function testDashboardPageLoads()
+     * author : Wang Shen 
+     * @return void
+     */
+
+    public function testDashboardMyTasks_FinalRadiologicalReviews()
+    {
+        $this->webDriver->get($this->url . '?test_name=dashboard');
+
+        $siteAll = $this->webDriver
+            ->findElement(WebDriverBy::cssSelector("#lorisworkspace > div > div.col-lg-4 > div:nth-child(1) > div > div.panel-body > div > a.list-group-item.radiological-review > div > div.col-xs-4.text-right.alert-chevron > span"));
+            $siteAll ->click();
+
+         $assertText1 = $this->webDriver
+            ->findElement(WebDriverBy::cssSelector("#page > div > div.alert.alert-info.alert-sm > a > label"))->getText();
+          
+        $this->assertContains("Final Radiological Review", $assertText1);
+        
+        
+    }
+
+
+    /**
+     * Tests that, when loading the Dashboard, click the stie all button of Accounts pending approval in the My tasks panel, it relocates to Mri Violations.
+     * author : Wang Shen 
+     * @return void
+     */
+
+    public function testDashboardMyTasks_AccountsPendingApproval()
+    {
+        $this->webDriver->get($this->url . '?test_name=dashboard');
+
+        $siteAll = $this->webDriver
+            ->findElement(WebDriverBy::cssSelector("#lorisworkspace > div > div.col-lg-4 > div:nth-child(1) > div > div.panel-body > div > a.list-group-item.pending-accounts > div > div.col-xs-4.text-right.alert-chevron > span"));
+            $siteAll ->click();
+
+         $assertText1 = $this->webDriver
+            ->findElement(WebDriverBy::cssSelector("#page > div > div.alert.alert-info.alert-sm > a > label"))->getText();
+           
+        $this->assertContains("User Accounts", $assertText1);
+        
+        
+    }
+
+    /**
+     * Tests that, when loading the Dashboard, click Document Repository, it relocates to Document Repository.
+     * author : Wang Shen 
+     * @return void
+     */
+
+    public function testDashboardMyTasks_DocumentRepository()
+    {
+        $this->webDriver->get($this->url . '?test_name=dashboard');
+
+        $siteAll = $this->webDriver
+            ->findElement(WebDriverBy::cssSelector("#lorisworkspace > div > div.col-lg-4 > div:nth-child(2) > div > div.panel-body > a"));
+            $siteAll ->click();
+
+         $assertText1 = $this->webDriver
+            ->findElement(WebDriverBy::cssSelector("#page > div > div.alert.alert-info.alert-sm > a > label"))->getText();
+           
+           
+        $this->assertContains("Document Repository", $assertText1);  
+        
+    }
+
+    /**
+     * Tests that, when loading the Dashboard as admin account, the submenu configuration should be appeared.
+     * author : Wang Shen 
+     * @return void
+     */
+
+  public function testDashboard_GetPermossion()
+  {
+        
+    $this->webDriver->get($this->url . '?test_name=dashboard');
+    $this->webDriver->get("https://wangshen-dev.loris.ca//configuration/");
```

Comment:
Yes. Next time I will fix it, Thanks.

2016-02-01 10:44 GMT-05:00 Ted Strauss <notifications@github.com>:

> In modules/dashboard/test/DashboardTest.php
> <https://github.com/aces/Loris/pull/1515#discussion_r51433750>:
>
> > +
> > +        $this->assertContains("Document Repository", $assertText1);
> > +
> > +    }
> > +
> > +    /**
> > +     * Tests that, when loading the Dashboard as admin account, the submenu configuration should be appeared.
> > +     * author : Wang Shen
> > +     * @return void
> > +     */
> > +
> > +  public function testDashboard_GetPermossion()
> > +  {
> > +
> > +    $this->webDriver->get($this->url . '?test_name=dashboard');
> > +    $this->webDriver->get("https://wangshen-dev.loris.ca//configuration/");
>
> static URL probably not a good idea.
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/aces/Loris/pull/1515/files#r51433750>.
>